### PR TITLE
Low hanging fruit tests for coverage.

### PR DESCRIPTION
**Pull request by surfnerd**
### Proposed change(s)

Add extra tests to boost coverage.

### Types of change(s)

- [x] Other (please describe) - Read above.

### Checklist
- [x] Added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs [hunk 1]
```diff
@@ -19,8 +175,7 @@ public void SetUp()
         {
             if (Academy.IsInitialized)
             {
-                Academy.Instance.Dispose();
-            }
+                Academy.Instance.Di   se            }
```
Comment (chriselion) [inline com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs:178]: Whoa, what happened there?

File: com.unity.ml-agents/Runtime/Policies/RemotePolicy.cs [hunk 2]
```diff
@@ -17,7 +17,9 @@ internal class RemotePolicy : IPolicy
         string m_FullyQualifiedBehaviorName;
         ActionSpec m_ActionSpec;
         ActionBuffers m_LastActionBuffer;
+#if MLA_UNITY_ANALYTICS_MODULE
```
Comment (chriselion) [inline com.unity.ml-agents/Runtime/Policies/RemotePolicy.cs:20]: Is this necessary for improving coverage?

Comment (surfnerd) [reply com.unity.ml-agents/Runtime/Policies/RemotePolicy.cs:20]: probably a merge mistake

File: com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs (hunk 1 above)
Comment (surfnerd) [reply com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs]: uhhh, let's not talk about it ;)

**Review by chriselion — APPROVED** [review]

**Pull request merged**